### PR TITLE
chore: Add replace_insecure_js zone_settings_override toggle.

### PR DESCRIPTION
**Pull request by anordby**

### CODE REVIEW

**Comment by jacobbednarz** [comment]
if we're adding a new field, we need to add an acceptance test for coverage. as is, i also don't think this works given the new property isn't in the aggregated view and needs to be treated as a single endpoint fetch as we did for `nel` in #3305.

File: internal/sdkv2provider/schema_cloudflare_zone_settings_override.go [hunk 1]
```diff
@@ -784,6 +791,13 @@ var resourceCloudflareZoneSettingsSchemaV0 = map[string]*schema.Schema{
 		Computed:     true,
 	},
 
+	"replace_insecure_js": {
```
[the remark this answers is not in the document]
Comment (anordby) [reply internal/sdkv2provider/schema_cloudflare_zone_settings_override.go]: Ok I removed the ones in V0/V1 @jacobbednarz

**Comment by anordby** [comment]
> if we're adding a new field, we need to add an acceptance test for coverage. as is, i also don't think this works given the new property isn't in the aggregated view of settings and needs to be treated as a single endpoint fetch as we did for `nel` in #3305.

Added a test, but seems it is not run from this PR? How do I test it manually? @jacobbednarz

**Comment by jacobbednarz** [comment]
you'll need to run the acceptance tests locally - https://github.com/cloudflare/terraform-provider-cloudflare/blob/master/contributing/development.md#quick-start

PRs only trigger unit tests as the acceptance tests take a long time and use real resources.

**Comment by anordby** [comment]
I was able to run the acceptance test for my test locally with a dummy Cloudflare account with some "pending" domains finally, and after setting a couple of defaults (challenge ttl and security level)) it completes successfully:

`TESTARGS='-run "^TestAccCloudflareZoneSettingsOverride_ReplaceInsecureJS" -count 1 -parallel 1' make testacc
(..)
ok      github.com/cloudflare/terraform-provider-cloudflare/internal/framework/service/zero_trust_access_mtls_hostname_settings                      0.403s [no tests to run]
testing: warning: no tests to run
PASS
ok      github.com/cloudflare/terraform-provider-cloudflare/internal/framework/service/zero_trust_risk_behavior 0.395s [no tests to run]
testing: warning: no tests to run
PASS
ok      github.com/cloudflare/terraform-provider-cloudflare/internal/framework/service/zero_trust_risk_score_integration                             0.389s [no tests to run]
=== RUN   TestAccCloudflareZoneSettingsOverride_ReplaceInsecureJS
--- PASS: TestAccCloudflareZoneSettingsOverride_ReplaceInsecureJS (26.39s)
PASS
ok      github.com/cloudflare/terraform-provider-cloudflare/internal/sdkv2provider      26.784s
testing: warning: no tests to run
PASS
ok      github.com/cloudflare/terraform-provider-cloudflare/internal/utils      0.362s [no tests to run]
`

Does everything look OK then for you to consider adding this toggle? @jacobbednarz

**Comment by jacobbednarz** [comment]
acceptance tests all looking good here. 

```
TF_ACC=1 go test ./internal/sdkv2provider -v -run "^TestAccCloudflareZoneSettingsOverride_" -count 1 -timeout 120m -parallel 1
=== RUN   TestAccCloudflareZoneSettingsOverride_Full
--- PASS: TestAccCloudflareZoneSettingsOverride_Full (31.81s)
=== RUN   TestAccCloudflareZoneSettingsOverride_RemoveAttributes
--- PASS: TestAccCloudflareZoneSettingsOverride_RemoveAttributes (39.40s)
=== RUN   TestAccCloudflareZoneSettingsOverride_NEL
--- PASS: TestAccCloudflareZoneSettingsOverride_NEL (13.29s)
=== RUN   TestAccCloudflareZoneSettingsOverride_ReplaceInsecureJS
--- PASS: TestAccCloudflareZoneSettingsOverride_ReplaceInsecureJS (14.28s)
PASS
ok  	github.com/cloudflare/terraform-provider-cloudflare/internal/sdkv2provider	100.203s
```

thanks!

**Pull request merged**